docs/superpowers/plans/2026-05-04-rob-103-watch-order-intent-mvp-finish.md

```
@@ -0,0 +1,761 @@
+| `56d6e511` | feat(ROB-103): add watch order intent preview builder |
+| `9b82a90a` | feat(ROB-103): add WatchOrderIntentLedger ORM model |
+| `b2dce522` | chore(ROB-103): add watch_order_intent_ledger migration |
+| `e85f31be` | feat(ROB-103): add watch order intent ledger service |
+| `f485674e` | feat(ROB-103): extend watch alerts service for action policies |
+| `c87500fa` | feat(ROB-103): add intents block to n8n watch alert payload |
+| `e37fce2f` | feat(ROB-103): branch watch scanner to intent service |
+| `36232080` | feat(ROB-103): extend manage_watch_alerts MCP tool for intents |
+
+**Uncommitted in working tree (Task 10 — written, but with bugs):**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)